pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -217,13 +218,17 @@ func getCluster(ctx context.Context, path string, finder *find.Finder, client *v
 		return "", errors.New("did not find any clusters")
 	}
 	if len(clusters) == 1 {
-		logrus.Infof("Defaulting to only available cluster: %s", clusters[0].Name())
-		return clusters[0].Name(), nil
+		name := strings.TrimLeft(clusters[0].InventoryPath, path)
+		name = strings.TrimLeft(name, "/host/")
+		logrus.Infof("Defaulting to only available cluster: %s", name)
+		return name, nil
 	}
 
 	var clusterChoices []string
 	for _, c := range clusters {
-		clusterChoices = append(clusterChoices, c.Name())
+		name := strings.TrimLeft(c.InventoryPath, path)
+		name = strings.TrimLeft(name, "/host/")
```

Comment:
nit: I think `[TrimPrefix](https://golang.org/pkg/strings/#TrimPrefix)` might be more appropriate here (and elsewhere), and the trim could be consolidated into a single call.

Reply:
This **must** use `TrimPrefix` instead. Otherwise, you will end up trimming more than you intend to. For example, if the name is something like `/host/hello`, it will trim down to `ello` instead of `hello`.

https://play.golang.org/p/Xf8oto1noG7

Reply:
That explains a lot! Thanks. Fixed in https://github.com/openshift/installer/commit/0afab1b4bcac64773a266242d563cc7bb94fde50